.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
I forgot to mention, we wanna rename `main` to `master` everywhere in this file since the repo was setup with `master` being the main branch.

Reply:
or the other way around? rename the branch from `master` to `main`. If I'm not mistaken github has a feature to easily do that